web-app/src/main/webapp/css/style.css

```diff
@@ -1,9 +1,8 @@
-/* Home screen */
-.report,
-.menu {
+.report {
```

Comment:
Maybe this is not needed here because we currently hide the report anyway on page load on the js side.

Reply:
I'll get rid of it once we merge everything together!